libs/contract/src/lib/negotiation/pipe.ts

```diff
@@ -0,0 +1,43 @@
+import { NgModule, Pipe, PipeTransform } from "@angular/core";
+import { OrganizationQuery, OrganizationService } from "@blockframes/organization/+state";
+import { of } from "rxjs";
+import { map } from "rxjs/operators";
+import { Negotiation } from "./+state/negotiation.firestore";
+
+@Pipe({ name: 'negotiationStage' })
+export class NegotiationStagePipe implements PipeTransform {
+  activeOrgId = this.orgQuery.getActiveId();
+
+  constructor(
+    private orgQuery: OrganizationQuery,
+    private orgService: OrganizationService,
+  ) { }
+
+  //here orgName is the seller that we need to get from stakeholders that are neither buyerId nor sellerId)
+  getOrgName(negotiation: Negotiation) {
+    const excluded = [negotiation.buyerId, negotiation.sellerId]
+    const seller = negotiation.stakeholders.find(s => !excluded.includes(s))
+    return seller;
+  }
+
+  transform(negotiation: Negotiation, ...args: any[]): any {
```

Comment:
Remove `...args` you don't need them
```suggestion
  transform(negotiation: Negotiation): Observable<string> {
```